lib/flashcard_runner.rb

```diff
@@ -0,0 +1,15 @@
+require './card'
+require './turn'
+require './deck'
+require './round'
+
+card_1 = Card.new("What is 5 + 5", "10", :STEM)
+card_2 = Card.new("What is Rachel's favorite animal", "Red Panda", :"Turing_Staff")
```

Comment:
Careful - your symbol is not formatted correctly. You meant to do `:Turing_Staff` I think?